### ISSUE TITLE
[Accessibility] Non-alphabetic character input is not bypassed in terminal and Copilot Chat accessible view

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#197250]
@meganrogge

I think I have found the issue. If you start typing non-alphabetic character in either terminal or Copilot Chat response Accessible View, you will see the alert: "Cannot edit in read-only editor."

I, as a bilingual user, sometimes need to type in Korean. Current implementation only bypasses alphabetic letters.

**Comment by meganrogge** [comment]
@jooyoungseo I just tried this and it worked on macOS:

- install Japanese, set it as the keyboard
- focus chat reply
- open accessible view
- type は
- the chat input is focused and contains は

**Comment by meganrogge** [comment]
I just tried Korean with this character and it worked ㄹ

**Comment by meganrogge** [comment]
arabic works on windows for me, currently installing Korean

**Comment by meganrogge** [comment]
when I switch to Korean and type, it's still producing English characters. 

@Tyriar speaks some Korean/ has it configured correctly. Can you repro?

**Comment by jooyoungseo** [comment]
After you add Korean keyboard set on Windows, you first have to press left alt+shift key to change to Korean keyboard set, and then, you can switch between English and Korean by pressing right Alt key.

**Comment by meganrogge** [comment]
Thanks, I can reproduce. I'm seeing the characters actually get inserted in the readonly editor 🐛 erroneously.

**Comment by meganrogge** [comment]
![image](https://github.com/microsoft/vscode/assets/29464607/5722b56c-fa28-4281-8c55-3aa2f296e349)

**Comment by roblourens** [comment]
Works as expected for Korean, but it's just checking a subset of characters I might type, for example `[]`, all shift-combos, and likely non-english keys on international keyboards will not hide the view, is that a problem?

**Comment by meganrogge** [comment]
That's by design @roblourens. We are checking if it's just a single letter key, which indicates someone's just inputting

**Comment by meganrogge** [comment]
Re non English keys on international keyboards, idk, that might work?

**Comment by roblourens** [comment]
> which indicates someone's just inputting

But not if I'm inputting the other characters I listed?
